src/main/java/com/rackspace/salus/monitor_management/web/controller/MonitorApiController.java

```diff
@@ -344,4 +344,10 @@ public void delete(@PathVariable String tenantId,
     return PagedContent.fromPage(monitorManagement.getMonitorsBySearchString(tenantId, searchCriteria, page)
       .map(monitor -> monitorConversionService.convertToOutput(monitor)));
   }
+
+  @DeleteMapping("/admin/tenant/{tenantId}")
```

Comment:
It looks like this URL path will clash with the zones one. Probably needs to be
```suggestion
  @DeleteMapping("/admin/tenant/{tenantId}/monitors")
```